README.md

```diff
@@ -426,11 +373,7 @@ const optimizely = createInstance({
 });
 ```
 

```

Comment:
Line 271: please update [Optimizely X Full Stack developer docs site](https://docs.developers.optimizely.com/full-stack/docs) to [Optimizely Feature Experimentation developer docs site](https://docs.developers.optimizely.com/experimentation/v4.0.0-full-stack/docs/welcome)